backend/apps/owasp/templates/search/issue.html

```diff
@@ -155,6 +155,21 @@ <h5>How to tackle it</h5>
                     </div>
                 </div>
             </div>
+            <!-- Add Pagination Component -->
+            <div class="pagination">
+                <!-- Left Button: Decrement page -->
+                <button class="pagination-button"
+                        v-model="page"
+                        @click="decrementPage"
+                        :disabled="page === 0">Previous</button>
+                <!-- Page and Total Pages Display -->
+                <span>${this.page} / ${this.totalPages}</span>
+                <!-- Right Button: Increment page -->
+                <button class="pagination-button"
```

Comment:
Some more improvement suggestions:
  - after I search for any term the pagination doesn't get reset and still shows the same page number instead of page 1
  - the number of available pages is always the same (in my case 251) -- that's the total page count for the case when you have an empty search query. I think you should use [nbPages](https://www.algolia.com/doc/api-reference/api-parameters/page/#usage-notes) from Algolia API response 
 
![Screenshot 2024-11-05 at 5 39 17 PM](https://github.com/user-attachments/assets/7a10eeb6-c3d9-4465-aa15-81a027f10182)